docs/IR.md

```diff
@@ -115,7 +115,7 @@ Name|Type|Description
 magic|string|The value ‘ONNXOPSET’
 ir_version|int32|The ONNX version corresponding to the operators.
 ir_version_prerelease|string|The prerelease component of the SemVer of the IR.
-ir_build_metadata|string|The symbolic identifier of the operator to invoke.
+ir_build_metadata|string|TODO come up with wording here
```

Comment:
the original looks like a copy-paste error. need some help with the actual wording

Reply:
The build metadata of this version of the operator set?

Reply:
thanks for the suggestion.
since the field is named `ir_build_metadata`, is it the metadata of the operator set or the IR?
if the former, would a better name for this field be `opset_build_metadata`?

Reply:
Changing field names is tough, right? May be @linkerzhang can comment on it, but it may not be worth changing field names. (My suggestion was based on the documentation in the proto file … I don't really know what this is used for.)